### PR TITLE
Reference BigQueryStorageV1 api

### PR DESCRIPTION
**Pull request by Gaurangi94**

### CODE REVIEW

**Comment by Gaurangi94** [comment]
Upon invoking spark shell and reading data frames in Arrow format I am getting dependency error. Raising this PR in parallel to catch any obvious errors. @davidrabinowitz

File: connector/src/main/scala/com/google/cloud/spark/bigquery/direct/DirectBigQueryRelation.scala [hunk 1]
```diff
@@ -130,13 +127,14 @@ private[bigquery] class DirectBigQueryRelation(
...
-            .setRequestedStreams(maxNumPartitionsRequested)
-            .setReadOptions(readOptions)
-            .setTableReference(actualTableReference)
+            .setReadSession(ReadSession.newBuilder()
+              .setDataFormat(readDataFormat)
+              .setReadOptions(readOptions)
+              .setTable(actualTablePath
+            ))
+            .setMaxStreamCount(maxNumPartitionsRequested)
             // The BALANCED sharding strategy causes the server to assign roughly the same
```
Comment (Gaurangi94) [inline connector/src/main/scala/com/google/cloud/spark/bigquery/direct/DirectBigQueryRelation.scala:136]: Sharding Strategy is not available in v1 API. Is there any other known way to achieve this functionality? @davidrabinowitz

Comment (davidrabinowitz) [reply connector/src/main/scala/com/google/cloud/spark/bigquery/direct/DirectBigQueryRelation.scala:136]: The v1 API has removed the sharding strategy.

**Pull request merged**